rover-server/internal/mapper/rover/in/exposure.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2025 Deutsche Telekom IT GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package in
```

Comment:
Perhabs moving the packages to a subfolder such as "api" would help for the future, once we also need to map EventExposures and FileExposures.